themes/loop/css/styles.css

```diff
@@ -1217,6 +1222,7 @@ label {
     -ms-box-sizing: border-box;
     box-sizing: border-box;
     word-wrap: break-word;
+    *behavior: url(url("../lib/boxsizing.htc"));
```

Comment:
url() kaldes to gange flere steder. Er dette meningen? 
Eksempel:
`
*behavior: url(url("../lib/boxsizing.htc"));
`